reporting/tests/report.py

```diff
@@ -220,25 +221,28 @@ def do_task(url, task, kwargs={}):
         )
 
         ## EVAL STEP 4
-        ground_truth_path = f"scenarios/{scenario}/ground_truth/model_card.json"
-        if os.path.exists(ground_truth_path):
-            logging.info(f"Accuracy for {scenario}:{task}")
-            generated_card = json.dumps(result["result"]["job_result"]["card"])
-            with open(ground_truth_path) as file:
-                truth = file.read()
-                files = {
-                    "test_json_file": generated_card,
-                    "ground_truth_file": truth,
-                }
-                eval = requests.post(
-                    f"{MIT_TR_URL}/evaluation/eval_model_card",
-                    params={"gpt_key": OPENAI_API_KEY},
```

Comment:
what all changed here?

Reply:
Additional error handling for an error I encountered.